Sources/Tracking/Service/Request/TrackRequestBody.swift

```diff
@@ -2,7 +2,13 @@ import Foundation
 
 /// https://customer.io/docs/api/#operation/track
 internal struct TrackRequestBody<T: Encodable>: Encodable {
+    let type: EventType
     let name: String
     let data: T?
     let timestamp: Date?
 }
+
+public enum EventType: String, Codable {
```

Comment:
Since this is `Codable` and we allow customers to pass in a customer `jsonEncoder` for tracking screen views, we will need to explicitly define the `CodingKeys` for these Strings to make sure the API accepts the request.